### DISCUSSION TITLE
Allow for dragging the command palette from anywhere on the top bar

### DISCUSSION THREAD

**Discussion by YousufSSyed** [johnlindquist/kit#1411] [Ideas]
I can drag from the left side of the text bar but not the right where the buttons are, I want to be able to drag by clicking on blank space anywhere on the top bar.

**Comment by johnlindquist** [comment]
Which operating system are you using? Dragging works on mac and windows. Linux doesn't provide events for supporting some window management features

**Reply by YousufSSyed** [reply]
Oh I figured it out, it doesn't drag from the right side where the buttons are, even when I try dragging from the gaps there. I revised the title and body.
